lightning/web-app.md

```diff
@@ -203,7 +203,7 @@ npm WARN deprecated apollo-reporting-protobuf@3.4.0: The `apollo-reporting-proto
 npm WARN deprecated apollo-server-core@3.12.0: The `apollo-server-core` package is part of Apollo Server v2 and v3, which are now deprecated (end-of-life October 22nd 2023). This package's functionality is now found in the `@apollo/server` package. See https://www.apollographql.com/docs/apollo-server/previous-versions/ for more details.
 (#################⠂) ⠧ reify:value-or-promise: timing reifyNode:node_modules/foreground-child/node_modules/signal-exit Completed in 39393ms
 [...]
-> thunderhub@0.13.19 prepare
+> thunderhub@0.13.21 prepare
```

Comment:
This output is only an "Example of expected output", not need modification

Reply:
Yes, I know, but isn't better to display the same version of the one the user is installing? Just to make him feel sure that he is installing everything correctly.

Reply:
I wrote the word "Example" to specify that it was just an output example, and to ease the update guide process